pydantic_settings/__init__.py

```diff
@@ -1,6 +1,6 @@
 import warnings
 
-from .main import BaseSettings
+from .settings import BaseSettings
```

Comment:
What's the rationale behind renaming `main` to `settings`? I'd prefer that we kept the old name, also if we do rename, let's do so in a separate PR so the other changes in the file are easier to read.

Reply:
Nothing serious TBH, just semantics. Will change!